server/controllers/availability.js

```diff
@@ -0,0 +1,88 @@
+const Availability = require("../models/Availability");
+const asyncHandler = require("express-async-handler");
+const mongoose = require("mongoose");
+
+// @route POST /availability
+// @desc create a schedule
+// @access private
+exports.createSchedule = asyncHandler(async (req, res, next) => {
+    const schedule = req.body.schedule
+   
+    let newSchedule = new Availability({
+      profileId: schedule.profileId,
```

Comment:
As opposed to sending the profile id in the request body, let's obtain it from the logged in user (that way only the logged in user can create a schedule for themselves)

Reply:
not sure I follow? How do I have access to that info from my controller

Reply:
ah, does the user get passed when we include the credentials header?

Reply:
@natey37 I must've missed your comment above. We can access the logged in users' id from the request object - `req.user.id` - which we will use to query the Profile model and find the relevant profile